src/phoenix/experimental/evals/templates/template.py

```diff
@@ -91,17 +91,17 @@ def normalize_template(template: Union[PromptTemplate, str]) -> PromptTemplate:
     Returns:
         PromptTemplate: The normalized template.
     """
-    normalized_template = template
-    if not (isinstance(template, PromptTemplate) or isinstance(template, str)):
-        raise TypeError(
-            "Invalid type for argument `template`. Expected a string or PromptTemplate "
-            f"but found {type(template)}."
-        )
+    if isinstance(template, PromptTemplate):
+        return template
+
     if isinstance(template, str):
         try:
             normalized_template = PromptTemplate(text=template)
+            return normalized_template
         except Exception as e:
             raise RuntimeError(f"Error while initializing the PromptTemplate: {e}")
```

Comment:
This `try` / `except` is not doing much. Not clear the error message is going to be clearer to the user than the default error message from a failed prompt template initialization.

Reply:
Agreed completely